### ISSUE TITLE
Mapillary won't run without google play services

### ISSUE THREAD

**Issue by de-vries** [mapillary/mapillary_issues#1399]
The most recent version of the app (android) gives me a warming every time I go to the camera that "Mapillary won't run without google play services". The app works fine when ignoring the warning, but it's a pretty annoying one.

For me it's a conscious choice to not use google services so this warning is not going to change my mind.

**Comment by peterneubauer** [comment]
Agreed. Would you like to get that warning once as a choice, then disable it and go with the Bare bones api instead? Thinking about adding that choice.

**Comment by de-vries** [comment]
I don't mind if I get the warning once. so a "don't show this message again" option would be a good solution.

atm I'm using a Jolla phone and I use the app via the android compatibility layer, so, besides that I don't want it, I have not much reason to install google services. 
Before I had a android device and I had to go trough quite some trouble to make the phone into something usable without installing google play. So my guess is that everyone who uses android without google services does this on purpose.

Also, I wonder why the app would need google services.

**Comment by peterneubauer** [comment]
Hi there,
the 1.43 should now ask you and keep that pref. GPS is better when it comes to mixing different providers like WIFI etc.

**Comment by polarbearing** [comment]
What is the relation between google play and google wifi positioning? 
I cannot see such setting in 1.43.
I typically have my android location access set to GPS=yes, wifi/mobile=no, and never got such warning.

Why would a location fix, for mapillary purposes, be better when other location providers are on? If I search the next cafe and have GPS reflections, this combination might give me a higher reliability to be at a certain location, but I doubt that it helps making a mapillary image more precise.

**Comment by dekarl** [comment]
Mind to add a second option to the "install Google Play Services" popup for Blackberry 10 users installing the APK from the Play Store via Snap? The application works well. But the popup just does not work. (nothing meaningful happens when agreeing to install the Google Play Services) So an option to "disable Google Play Services" would be appreciated.
On 1.67 there was no such option. Instead get popup gets dismissed and reappears over and over. Manually turning Google Play Services off in the settings worked.

**Comment by utack** [comment]
Same problem for me in the newest "306" and "305" releases, plus "240"
Popup about Google Play right after starting, in the login screen, then a crash  
The odd thing is, I think the 240 release used to work on Android 7.0, but now crashes too.
Please remove the dependancy, I can't take photos with the mapillary app at the moment 👎

**Comment by polarbearing** [comment]
We were promised a lean, capture-only app a while ago: #1416. As the web uploader accepts pictures from the app (which in the past was not the case), my workflow is anyway to transfer the pictures to the laptop for large-screen review. I don't need any feed or upload information on the mobile device, thus we could get rid of all that troublesome parts.

**Comment by ovivoda** [comment]
_mapillary_issues is no longer an active repository._

_Let’s continue discussing features in our forum! We will import this issue as topic to the Mapillary Forum in the coming weeks. We’ll notify here once it’s been done. See you there!_
